docs/source/posts/2021/2021-06-08-week-1-antriksh.rst

```diff
@@ -0,0 +1,27 @@
+Week #1: Welcome to my weekly Blogs!
+====================================
+
+.. post:: June 08 2021
+   :author: Antriksh Misri
+   :tags: google
+   :category: gsoc
+
+Hi everyone! I am **Antriksh Misri**. I am a Pre-Final year student at MIT Pune. This summer, I will be working on **Layout Management** under FURY's UI module as my primary goal. This includes addition of different classes under Layout Management to provide different layouts/arrangements in which the UI elements can be arranged. As my stretch goals, I will be working on a couple of UI components for FURY’s UI module. These 2D and 3D components will be sci-fi like as seen in the movie “**Guardians of The Galaxy**”. My objective for the stretch goals would be to develop these UI components with their respective test and tutorials such that it adds on to the UI module of FURY and doesn’t hinder existing functionalities/performance.
+
+What did I do this week?
+------------------------
+During the community bonding period I got to know the mentors as well as other participants. We had an introductory meeting, in which the rules and code of conduct was explained. Also, my proposal was reviewed and modified slightly. Initially, I had to develop UI elements as my primary goal and I had to work on layout management as my stretch goals but the tasks were switched. Now I have to work on Layout Management as my primary task and develop UI in the stretch goals period. I also started coding before hand to actually make use of this free period. I worked on different PR's which are described below:-
```

Comment:
Can you remove the `-` at the end of the sentence

Reply:
Done.